client/src/pages/Plan/Header.js

```diff
@@ -75,7 +75,7 @@ const Header = ({ name, programme, lastSaveDate, planName, setPlanName, optionsM
 
                         <Flex height="50px" backgroundColor={c.lightBlue} borderRadius="10px" align="center" justify="center" p={5} mt={2}>
                             <Text textAlign="center" color={c.white} fontSize="xl">
-                                {programme ? programme : "Program could not be found"}
+                                {programme || "Program could not be found"}
```

Comment:
Should we make this

"Programme could not be found" instead of "Program could not be found"

Reply:
sgtm